### PR TITLE
nits for sync-subnet-evm

### PR DESCRIPTION
**Pull request by darioush**

### CODE REVIEW

File: ethclient/corethclient/corethclient.go [hunk 1]
```diff
@@ -186,7 +186,7 @@ func toCallArg(msg interfaces.CallMsg) interface{} {
 		"to":   msg.To,
 	}
 	if len(msg.Data) > 0 {
-		arg["data"] = hexutil.Bytes(msg.Data)
+		arg["input"] = hexutil.Bytes(msg.Data)
```
Comment (darioush) [inline ethclient/corethclient/corethclient.go:189]: highlighting this

File: params/network_upgrades.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		{name: "apricotPhase3BlockTimestamp", timestamp: c.ApricotPhase3BlockTimestamp},
+		{name: "apricotPhase4BlockTimestamp", timestamp: c.ApricotPhase4BlockTimestamp},
+		{name: "apricotPhase5BlockTimestamp", timestamp: c.ApricotPhase5BlockTimestamp},
+		{name: "apricotPhasePre6BlockTimestamp", timestamp: c.ApricotPhasePre6BlockTimestamp},
+		{name: "apricotPhase6BlockTimestamp", timestamp: c.ApricotPhase6BlockTimestamp},
+		{name: "apricotPhasePost6BlockTimestamp", timestamp: c.ApricotPhasePost6BlockTimestamp},
+		{name: "banffBlockTimestamp", timestamp: c.BanffBlockTimestamp},
+		{name: "cortinaBlockTimestamp", timestamp: c.CortinaBlockTimestamp},
+		{name: "durangoBlockTimestamp", timestamp: c.DurangoBlockTimestamp},
+		{name: "cancunTime", timestamp: c.CancunTime}, // XXX: should this be in ethForks?
```
Comment (darioush) [inline params/network_upgrades.go:19]: see comment, should this be in ethForks? I think this check is missing from subnet-evm

File: plugin/evm/atomic_syncer_test.go [hunk 3]
```diff
@@ -177,7 +177,6 @@ func TestAtomicSyncerResume(t *testing.T) {
 }
 
 func TestAtomicSyncerResumeNewRootCheckpoint(t *testing.T) {
-	t.Skip("FLAKY")
```
Comment (darioush) [inline plugin/evm/atomic_syncer_test.go:180]: this test should not be flaky in master

File: params/network_upgrades.go (hunk 2 above)
Comment (ceyonur) [reply params/network_upgrades.go:19]: yes I think it fits better in ethforks (as we indirectly activate it via `EUpgrade`). In Subnet-EVM there is a `SetEVMUpgrades` that should activate cancun. I put up a draft PR to add Eupgrade and map to Cancun here: https://github.com/ava-labs/subnet-evm/pull/1135/files.

**Review by ceyonur — APPROVED** [review]
lgtm

**Pull request merged**